fdbserver/SimulatedCluster.actor.cpp

```diff
@@ -1285,6 +1286,14 @@ void SimulationConfig::setStorageEngine(const TestConfig& testConfig) {
 		set_config("ssd-redwood-experimental");
 		break;
 	}
+	case 4: {
+		TEST(true); // Simulated cluster using RocksDB storage engine
+		set_config("ssd-rocksdb-experimental");
+		// Tests using the RocksDB engine are necessarily non-deterministic because of RocksDB
```

Comment:
Can you please add a Sev30 trace here that tells anyone looking at the traces that the result is non-deterministic? Otherwise I fear build cops will waste a lot of time trying to find non-determinism bugs (I understand it won't print an unseed, but having a second mention here would be useful).

Reply:
Done